app/util/network.py

```diff
@@ -38,7 +38,8 @@ def reset_session(self):
         if self._session:
             self._session.close()  # Close any pooled connections held by the previous session.
         self._session = requests.Session()
-        self._session.mount('http://', HTTPAdapter(pool_connections=self._poolsize, pool_maxsize=self._poolsize))
+        self._session.mount('{}://'.format(Configuration['protocol_scheme']),
```

Comment:
Should `protocol_scheme` be a constant? It's used in several places across multiple files.